# Control commands for the AscTec HLP/LLP
# Expect rpyt to be in -1 .. +1 double
# ctrl_mode is 8 bits: [ x x gps height thrust yaw roll pitch ]

float64 roll
float64 pitch
float64 yaw
float64 thrust
uint8 ctrl_mode
